nsxt/utils_test.go

```diff
@@ -445,6 +445,19 @@ resource "nsxt_policy_tier%s_gateway" "test" {
 }`, tier, tier, edgeClusterName, haMode)
 }
 
+func testAccNsxtPolicyTier0WithEdgeClusterForVPN(edgeClusterName string) string {
+	// VPN is supported only on Tier-0 with ACTIVE-STANDBY HA mode
+	return fmt.Sprintf(`
+resource "nsxt_policy_tier0_gateway" "test" {
+  display_name = "terraform-t0-gw-for-vpn"
```

Comment:
It is preferrable to use `getAccTestResourceName` to avoid collisions between different tests running simultaneously.